### PR TITLE
Improved support for Codeception 5/PHP 8

### PR DESCRIPTION
**Pull request by yesdevnull**
I still need to do the following:

- [x] Consolidate `composer.json` changes with other Codeception modules.
- [x] Convert stray tabs to spaces (or at least consistent with the current codebase).
- [x] Correct commit message to be consistent with repository.

Bonus points (for a separate PR maybe):
- [ ] PHPStan (using config from [`module-doctrine2`](https://github.com/Codeception/module-doctrine2/blob/master/phpstan.neon))

### CODE REVIEW

**Review by yesdevnull — COMMENTED** [review]
Same comment re `composer.json` as https://github.com/Codeception/module-symfony/pull/154#pullrequestreview-877179332.

**Comment by TavoNiievez** [comment]
LGTM so far.

File: src/Codeception/Module/DataFactory.php [hunk 1]
```diff
@@ -194,7 +200,7 @@ public function _beforeSuite($settings = [])
         $this->factoryMuffin = new FactoryMuffin($store);
 
         if ($this->config['factories']) {
-            foreach ((array)$this->config['factories'] as $factoryPath) {
+            foreach ((array) $this->config['factories'] as $factoryPath) {
```
Comment (TavoNiievez) [inline src/Codeception/Module/DataFactory.php]: about this little space... Normally PHPStorm and some linters suggest not to add spaces.
I recently merged a PR in Codeception ( https://github.com/Codeception/Codeception/pull/6353 ) using the version with no spaces throughout the code.

Comment (yesdevnull) [reply src/Codeception/Module/DataFactory.php]: That was my mistake, sorry! I'll revert and re-push.

Comment (TavoNiievez) [reply src/Codeception/Module/DataFactory.php:203]: I'm going to use the version without spaces before merging the 3.0 branch but please note that we use that style in your future PRs.

**Review by TavoNiievez — APPROVED** [review]

**Pull request merged**